manim/scene/scene_file_writer.py

```diff
@@ -68,26 +68,18 @@ def init_output_directories(self):
         """
         module_directory = self.output_directory or self.get_default_module_directory()
         scene_name = self.file_name or self.get_default_scene_name()
-        #print("1")
-        #print(dirs.MEDIA_DIR)
-        if self.save_last_frame or self.save_pngs:
-            if dirs.MEDIA_DIR != "":
-                image_dir = guarantee_existence(os.path.join(
-                    dirs.MEDIA_DIR,
-                    "images",
-                    module_directory,
-                ))
+        if self.save_last_frame:
+            image_dir = guarantee_existence(os.path.join(
+                dirs.VIDEO_DIR
+            ))
             self.image_file_path = os.path.join(
                 image_dir,
                 add_extension_if_not_present(scene_name, ".png")
             )
         if self.write_to_movie:
-            if dirs.VIDEO_DIR != "":
-                movie_dir = guarantee_existence(os.path.join(
-                    dirs.VIDEO_DIR,
-                    module_directory,
-                    self.get_resolution_directory(),
-                ))
+            movie_dir = guarantee_existence(os.path.join(
+                dirs.VIDEO_DIR
+            ))
```

Comment:
It is my understanding from #66 that we are still going to have subdirectories for different resolutions. CC @eulertour

Reply:
In my opinion subdirectories for different resolutions are not that beneficial, because with them it is not anymore easy to switch between videos when working with the explorer.
Further, it is very easy to obtain the video result on very quickly: right-click on the file, and then on "show information", or you can get a feeling of the resolution when looking on the size the video file has. Also, any video-editing software is showing the clip resolution.
So I would definitely prefer the simpler version, where videos are easy to access.

Reply:
I wouldn't prefer being forced to run with the simple structure. Making it optional would be okay, forcing it on everyone, not so much.